lib/sna/auth/email_supervisor.ex

```diff
@@ -0,0 +1,16 @@
+defmodule Sna.Auth.Mail.Supervisor do
```

Comment:
Shouldn't the file be named `lib/sna/aith/mail/supervisor.ex` instead?

Reply:
I will change that. typo.